src/ServicePulse.Host/vue/src/components/failedmessages/MessageView.vue

```diff
@@ -32,29 +37,29 @@ async function loadFailedMessage() {
   try {
     const response = await useFetchFromServiceControl("errors/last/" + id.value);
     if (response.status === 404) {
-      failedMessage.value = { notFound: true };
+      failedMessage.value = { notFound: true } as FailedMessageViewWithExtendedUIProperties;
     } else if (response.status !== 200) {
-      failedMessage.value = { error: true };
+      failedMessage.value = { error: true } as FailedMessageViewWithExtendedUIProperties;
     }
```

Comment:
- should be checking response.ok rather than status 200
- should this be returning after this line, rather than continuing to try and read response.json which is guaranteed to throw an exception?
- should the error/notFound values be on a different type, rather than typecasting an object that doesn't match the interface to FailedMessageViewWithExtendedetc? I realise this would then require 'is' checks where these values are read, but that would definitely be clearer and potentially cleaner